### ISSUE TITLE
pre-commit hook does not support filenames containing accents

### ISSUE THREAD

**Issue by glepretre** [Kozea/MakeCitron#15]
Steps to reproduce : 

* Edit jinja2 files (templates containing accents in filename for example : `à-propos.html.jinja2`
* git add .
* git commit -m "Test"

Expected Result : 

- Files are linted and formatted

Actual Result : 

```
    🞋  pre-commit 
Usage: djlint [OPTIONS] SRC ...
Try 'djlint -h' for help.

Error: Invalid value for 'SRC ...': Path 'www_kozea/templates/\\303\\240-propos.html.jinja2' does not exist.
make: *** [base.Makefile:146: pre-commit] Error 2
```

**Comment by glepretre** [comment]
Due to git diff quoting paths in this case : 

https://github.com/Kozea/MakeCitron/blob/c493a3286aa96e1ff5e104bf47d04efcde00e18f/base.Makefile#L135

Users may not hit the bug if they have enable `core.quotepath false` in git config.

Best way to fix is to use `-z` on git diff commands: https://git-scm.com/docs/git-diff#Documentation/git-diff.txt--z
Will apply to v5 (current) and v3-legacy branches.

Thanks @gentooboontoo for the help finding it :pray:

**Comment by glepretre** [comment]
Fixed by #16 and #17
